packages/grafana-prometheus/src/components/PrometheusMetricsBrowser.tsx

```diff
@@ -409,11 +409,15 @@ export class UnthemedPrometheusMetricsBrowser extends React.Component<BrowserPro
     }
   };
 
+  getTimeRange = (): TimeRange => {
+    return this.props.timeRange ?? getDefaultTimeRange();
+  };
+
   async fetchValues(name: string, selector: string) {
     const { languageProvider } = this.props;
     this.updateLabelState(name, { loading: true }, `Fetching values for ${name}`);
     try {
-      let rawValues = await languageProvider.getLabelValues(name);
+      let rawValues = await languageProvider.getLabelValues(this.getTimeRange(), name);
```

Comment:
Hi @itsmylife! Just wanted to point out that we were using this in Grafana Metrics Drilldown and changing the signature caused a bug. We always need to pass in the time range now I see! And handle when GMD is in older versions without this change in function parameters.

I see levitate noted it as a breaking change. Just wanted to point this out!